test/testinput/bufr_ncep_adpupa.yaml

```diff
@@ -0,0 +1,156 @@
+# (C) Copyright 2020 NOAA/NWS/NCEP/EMC
+# 
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+
+observations:
+  - obs space:
+      name: bufr
+
+      obsdatain: "./gdas.t12z.adpupa.tm00.bufr_d"
+
+      exports:
+        group_by_variable: pressure
+        variables:
+          # MetaData
+          timestamp:
+            datetime:
+              year: "*/YEAR"
+              month: "*/MNTH"
+              day: "*/DAYS"
+              hour: "*/HOUR"
+              minute: "*/MINU"
+          stationIdentification:
+            query: "*/RPID"
+          latitude:
+            query: "*/CLAT"
+          longitude:
+            query: "*/CLON"
+          stationElevation:
+            query: "*/SELV"
+
+          # ObsValue
+          pressure:
+            query: "[*/UARLV/PRLC]"
+          airTemperature:
+            query: "[*/UARLV/UATMP/TMDB]"
+          dewpointTemperature:
+            query: "[*/UARLV/UATMP/TMDP]"
+          windDirection:
+            query: "[*/UARLV/UAWND/WDIR]"
+          windSpeed:
+            query: "[*/UARLV/UAWND/WSPD]"
```

Comment:
Brackets are not necessary as they only contain 1 query.

Reply:
@rmclaren these changes have been implemented and [] were removed.

Reply:
@rmclaren fixed and updated.

Reply:
@rmclaren could you review this again?